agent/instrumentation/micrometer-1.0/src/main/java/io/opentelemetry/javaagent/instrumentation/micrometer/ai/AzureMonitorRegistryConfig.java

```diff
@@ -10,12 +10,18 @@
 public class AzureMonitorRegistryConfig implements StepRegistryConfig {
 
   private final Duration step;
+  private final String namespace;
 
-  public AzureMonitorRegistryConfig() {
+  public static final AzureMonitorRegistryConfig INSTANCE = new AzureMonitorRegistryConfig();
+
+  private AzureMonitorRegistryConfig() {
     step =
         InstrumentationConfig.get()
             .getDuration(
                 "applicationinsights.internal.micrometer.step.millis", Duration.ofSeconds(60));
+    namespace =
+        InstrumentationConfig.get()
+            .getString("applicationinsights.internal.micrometer.namespace", "");
```

Comment:
I think I'd suggest using `@Nullable` on namespace everywhere to make it more clear that it is optional

```suggestion
    namespace =
        InstrumentationConfig.get()
            .getString("applicationinsights.internal.micrometer.namespace");
```